ocean_provider/utils/encryption.py

```diff
@@ -1,15 +1,29 @@
 import ecies
+from ocean_lib.web3_internal.wallet import Wallet
 from web3 import Web3
 
 from ocean_provider.utils.accounts import get_private_key
 
 
 def do_encrypt(document, provider_wallet):
```

Comment:
using 2 params instead of the overloaded use of `provider_wallet` makes it more clear.